docs/DeveloperGuide.adoc

```diff
@@ -295,61 +295,147 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |trace all my internship application's contact | easily follow up on the application
 
-|`* * *` |user |add a new person |
+|`* * *` |user |tag each application with a status | track my internship application phase
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |self-reflecting user |mark what positions of internship I have been applying to | see which positions I have the best chance of getting and easily look up past internship application when applying to similar positions
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |set reminders for internship deadlines/appointments| make sure I do not miss any internship opportunities
```

Comment:
Would it be more accurate to describe this as "so that I can promptly complete the internship tasks", instead of "missing any internship opportunities", as missing internship opportunities imply not applying to it in the first place?